clang/test/CIR/CodeGen/types-IEEE-quad.c

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir --check-prefix=CHECK %s
+// RUN: cir-opt %t.cir -cir-to-llvm -o %t.mlir
```

Comment:
Please use `CIR` instead of `CHECK` and `LLVM` instead of `CHECK-MLIR`. For LLVM, please check the final output instead, see `clang/test/CIR/CodeGen/abstract-cond.c` for an example.

Reply:
Hey @bcardosolopes  , I am a bit confused about what you mean by final output?

Should I convert this llvm mlir to llvm ir as well ? In that case I would have to put CIR -> for clangir checking, LLVM-MLIR -> to check llvm mlir, LLVM -> to check llvm IR.

Reply:
I don't think you need to check the MLIR, just the CIR and the LLVM. See https://github.com/llvm/clangir/blob/main/clang/test/CIR/CodeGen/packed-structs.c for an example (which also demonstrates the check prefixes that were requested).